### ISSUE TITLE
Opposing Faction Name Plates

### ISSUE THREAD

**Issue by jander99** [Luxocracy/NeatPlates#260]
Hi! 

Thank you for your work on this addon. I'm currently trying to find the combinations of options that will allow me to set the opposing Faction's nameplates to be skinned the same as my faction plus NPCs. I haven't found a combo that works just yet. Is there such a feature? If not, how hard would it be to add that feature in an upcoming release? 

Thank you!

**Comment by Luxocracy** [comment]
I'm unsure of what issue you are facing in making the nameplates the same. Could you perhaps include an image of what your nameplates look like in these scenarios? Just to help me better understand what you are looking for.

**Comment by jander99** [comment]
Hi, sorry to leave this open so long! 

I was able to narrow down that the issue was with PvP-flagged player nameplates. I'm not sure if Blizzard is properly setting any flags that NeatPlates is able to read so the default styling is being applied.  I've since switched back to the original theme and I'm looking at moving to ElvUI and Plater. 

Closing this since I have nothing more to add, but I wanted to at least document that I think the PvP flag has something to do with what I saw. 

Thanks!
